.gitlab-ci.yml

```diff
@@ -22,6 +22,7 @@ stages:
 
 # Before executing a job's script, we check the output of docker info
 before_script:
+  - apk add --update bash
```

Comment:
Why do we need to update bash?

Reply:
It's explain on the PR message, it's for use the build script.

Reply:
Can you add a comment instead?

Reply:
I'm do this on the issue #59.
I've remove the commit for gitlab-ci.